File: src/main/java/com/lord/small_box/dtos/OrganizationDto.java
package com.lord.small_box.dtos;

import java.math.BigDecimal;
import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class OrganizationDto {

	private Long id;

	private String organizationName;

	private int organizationNumber;

	private String responsible;
	
	private Long responsibleId;

	private int currentRotation;

	private int maxRotation;

	private BigDecimal maxAmount;

}

File: front/small-box/src/app/models/organizationDto.ts
import { NumberSymbol } from "@angular/common";

export class OrganizationDto{
    id!:number;
    organizationName!:string;
    organizationNumber!:number;
    currentRotation!:number;
    maxRotation!:number;
    maxAmount!:number;
    responsible!:string;
    responsibleId!:number

}